### DISCUSSION TITLE
`labelmaker` - Create printable PDF labels from media directories

### DISCUSSION THREAD

**Discussion by sandreas** [MiczFlor/RPi-Jukebox-RFID#1530] [Show and tell]
Hello,

in the last days I worked on a little tool to automatically create labels for RFID stickers by scanning a media directory (e.g. your Jukebox folders). It is still in an early stage of development and I'm looking for testers and feedback.

Official documentation: 
https://pilabor.com/labelmaker/

Github repository: 
https://github.com/sandreas/labelmaker

Example usage:
```bash
labelmaker.phar --theme="labelmaker/phoniebox-audiobook/"  --output-file="labels.pdf" --data-uri="my-media-folder/" --data-records-per-page="10"
```

I would love to get some feedback.

Thank you
sandreas

**Comment by pabera** [comment]
This is pretty cool!

**Reply by sandreas** [reply]
Thx :-) I hope it's usable in newer releases of RPi-Jukebox-RFID itself, so it would be possible to download `labelmaker` on the raspberry and build printable label pdfs directly on it. My box is a bit older and since `PHP 7.4` is required, I cannot test this atm ;)
